.github/workflows/php-lint-test.yml

```diff
@@ -6,8 +6,9 @@ on:
 
 env:
   WP_VERSION:        latest
-  WC_L2_VERSION: '7.3.0'  # the min supported version as per L-2 policy
+  WC_MIN_VERSION: '7.3.0'
```

Comment:
The comment should stay IMO for clarity.

Reply:
Similar thoughts as https://github.com/Automattic/woocommerce-payments/pull/5924#discussion_r1155636653

Will update based on your reply